spec/system/report_request_for_comment_comments_system_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Report RfC comments for inappropriate content', :js do
```

Comment:
Wouldn't this be sufficient:
```suggestion
RSpec.describe 'Report Comments for inappropriate content', :js do
```

Reply:
I just wanted to be more specific because the comment is very unspecific. If we allow comments on other objects there would be a collision.

I have no problem rewriting this. In this case, I should change the file to `spec/system/report_comment_system_spec.rb` as well?